src/modules/dashboard/components/ActionsPage/DetailsWidget/DetailsWidgetUser/DetailsWidgetUser.css

```diff
@@ -0,0 +1,19 @@
+.main {
+  display: flex;
+  align-items: center;
+}
+
+.address {
```

Comment:
Please align the username with address so they are on the same line

![Screenshot from 2020-12-08 14-27-12](https://user-images.githubusercontent.com/1193222/101486368-3777a480-3965-11eb-91ba-e57dc47d38c4.png)